static/app/views/replays/replays.tsx

```diff
@@ -18,6 +19,7 @@ import {NewQuery, Organization, PageFilters} from 'sentry/types';
 import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
 import EventView from 'sentry/utils/discover/eventView';
 import {generateEventSlug} from 'sentry/utils/discover/urls';
+import Projects from 'sentry/utils/projects';
```

Comment:
Can you use the `useProjects` hook instead of this older render-props style utility.

ref https://github.com/getsentry/sentry/pull/33647

Reply:
Looks like `Replays` could be written as a functional component, in which case you could :)

But given how it's written now, not yet!